portal/patient/templates/_Header.tpl.php

```diff
@@ -14,7 +14,7 @@
 
 ?>
 <!DOCTYPE html>
-<html lang="en">
+<html>
     <head>
         <meta charset="utf-8" />
 
```

Comment:
`<?php Header::setupHeader(['no_main-theme', 'patientportal-style', 'opener', 'fontawesome', 'datetime-picker']); ?>`
Needs to be fixed. Portal hasn't anywhere to put 'opener'. remove.

Also, remove Lab script loads:
.script("<?php echo $GLOBALS['assets_static_relative']; ?>/jquery/dist/jquery.min.js").wait()
                .script("<?php echo $GLOBALS['assets_static_relative']; ?>/bootstrap/dist/js/bootstrap.bundle.min.js")
                .script("<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker/build/jquery.datetimepicker.full.min.js")

Otherwise we duplicate loading dependencies.

Reply:
Glad we're fixing this part 😂 the headers in all these templates are pretty terrible

Reply:
be careful with headers as there is a reason some are as they are.
There can be timing issues and loading order that is important.

Just leave these with me and i'll do follow up PR. There are other issues that I need to address as well.